features/support/alerts.rb

```diff
@@ -1,23 +1,18 @@
-# -*- coding: utf-8 -*-
-module Alerts
+# frozen_string_literal: true
 
-  def symbolize_headers(table)
-    table.map_headers! { |header| header.downcase.gsub(/\s+/, '_').to_sym }
-  end
```

Comment:
Although it has a single usage only I'd suggest moving it under `DataTableTransforms` and still use it in the step.

Reply:
Move to DataTableTransforms.